cluster/gce/configure-vm.sh

```diff
@@ -851,6 +851,16 @@ function node-docker-opts() {
   if [[ -n "${EXTRA_DOCKER_OPTS-}" ]]; then
     DOCKER_OPTS="${DOCKER_OPTS:-} ${EXTRA_DOCKER_OPTS}"
   fi
+
+  # Decide whether to enable the cache
+  if [[ "${ENABLE_DOCKER_REGISTRY_CACHE:-}" == "true" ]]; then
+    REGION=$(echo "${ZONE}" | cut -f 1 -d -)
```

Comment:
Where this env variable "${ZONE}" from? And it is better to use "${ZONE:-}" in case it is unset.